tools/tflitefile_tool/select_operator.py

```diff
@@ -63,7 +63,44 @@ def GetOperatorList(oplist_file):
     return opcode_list
 
 
-def GenerateOperatorCodes(new_builder, sample_model, used_operators_dic):
+def GetUsedSubgraphsList(sample_model, subg_num, opcode_list, used_subgraphs_list):
```

Comment:
@ragmani As I understand your PR, `opcode_list` here should be renamed to `operator_list` also.

Reply:
Right, I will change the name. Thanks for your pointing out.